From: Dimitri John Ledkov <dimitri.ledkov@canonical.com>
Date: Wed, 18 May 2022 08:58:57 -0400
Subject: Force enable tlsv1 for the tls1 tests.

Bug-Upstream: https://gitlab.com/m2crypto/m2crypto/-/issues/310
Bug-Debian: https://bugs.debian.org/1006508
---
 tests/test_ssl.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test_ssl.py b/tests/test_ssl.py
index e18adf5..01d63ff 100644
--- a/tests/test_ssl.py
+++ b/tests/test_ssl.py
@@ -409,6 +409,7 @@ class MiscSSLClientTestCase(BaseSSLClientTestCase):
             with warnings.catch_warnings():
                 warnings.simplefilter('ignore', DeprecationWarning)
                 ctx = SSL.Context('tlsv1')
+                ctx.set_cipher_list('DEFAULT@SECLEVEL=0')
             s = SSL.Connection(ctx)
             with six.assertRaisesRegex(self, SSL.SSLError,
                                        r'version|unexpected eof'):
